src/main/io/serial_cli.c

```diff
@@ -739,6 +741,46 @@ static void cliCMix(char *cmdline)
 #endif
 }
 
+static void cliRxCalibration(char *cmdline)
+{
+    int i, check = 0;
+    char *ptr;
+
+    if (isEmpty(cmdline)) {
+        for (i = 0; i < MAX_CALIBRATED_RX_CHANNELS; i++) {
+            printf("rccal %u %u %u\r\n", i, masterConfig.rxConfig.calibration[i].minrc, masterConfig.rxConfig.calibration[i].maxrc);
+        }
+    } else {
+        ptr = cmdline;
+        i = atoi(ptr);
+        if (i >= 0 && i < MAX_CALIBRATED_RX_CHANNELS) {
+            int minrc, maxrc;
+            ptr = strchr(ptr, ' ');
+            if (ptr) {
+                minrc = atoi(++ptr);
+                check++;
+            }
+            ptr = strchr(ptr, ' ');
+            if (ptr) {
+                maxrc = atoi(++ptr);
+                check++;
+            }
+            if (check != 2) {
```

Comment:
couldn't you just do `sizeof(ptr) != 2` ? and get rid of the check++ logic?

Other options: http://stackoverflow.com/questions/6150258/finding-length-of-char-array

just seems like extra work to increment a pointer to check number of arguments